packages/components/stories/label/label.stories.js

```diff
@@ -30,7 +29,7 @@ storiesOf('Label', module)
                 <Wrapper inner_styles={{}} is_dark={boolean('Dark Theme', false)}>
                     {available_modes.map((mode, index) => {
                         return (
-                            <div className={'container'}>
+                            <div className={'container'} key={index}>
```

Comment:
using index as key for mapping components is a anti pattern
please use mode for that it its unique